schemachange/cli.py

```diff
@@ -114,6 +116,22 @@ def schemachange(root_folder, snowflake_account, snowflake_user, snowflake_role,
         print("Skipping change script %s because it's older than the most recently applied change (%s)" % (script['script_name'], max_published_version))
       scripts_skipped += 1
       continue
+    
+    # Apply only R scripts where the checksum changed compared to the last execution of snowchange
+    if script_name[0] == 'R':
+      # get checksum of current R script
+      with open(script['script_full_path'],'r') as content_file:
+        content = content_file.read().strip()
+        content = content[:-1] if content.endswith(';') else content
+        checksum_current = hashlib.sha224(content.encode('utf-8')).hexdigest()
+        checksum_last = list(r_scripts_checksum.loc[r_scripts_checksum['script_name'] == script_name, 'checksum'])[0]
```

Comment:
This fails if the R__ script runs for the first time?

Reply:
Thanks for the review. I fixed this with the new commit.